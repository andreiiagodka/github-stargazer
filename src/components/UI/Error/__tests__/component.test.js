import { shallow } from 'enzyme'

import ErrorComponent from '..'

describe('<ErrorComponent />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ErrorComponent />)

    expect(wrapper).toMatchSnapshot()
  })
})
