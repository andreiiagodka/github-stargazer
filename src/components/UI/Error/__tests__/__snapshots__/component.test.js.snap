// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorComponent /> renders correctly 1`] = `
<Alert
  className="text-center"
  closeLabel="Close alert"
  show={true}
  transition={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "appear": false,
        "in": false,
        "mountOnEnter": false,
        "timeout": 300,
        "unmountOnExit": false,
      },
      "render": [Function],
    }
  }
  variant="danger"
>
  Something went wrong!
</Alert>
`;
